ara/clients/utils.py

```diff
@@ -22,7 +22,7 @@ def get_client(
     """
     auth = None
     if username is not None and password is not None:
-        auth = HTTPBasicAuth(username, password)
+        auth = HTTPBasicAuth(username.encode('utf-8'), password.encode('utf-8'))
```

Comment:
```suggestion
        auth = HTTPBasicAuth(username.encode("utf-8"), password.encode("utf-8"))
```
The linter is complaining because it wants double quotes.